package/debian/frankenphp.service

```diff
@@ -0,0 +1,27 @@
+# See https://caddyserver.com/docs/install for instructions.
+#
+# WARNING: This service does not use the --resume flag, so if you
+# use the API to make changes, they will be overwritten by the
+# Caddyfile next time the service is restarted. If you intend to
```

Comment:
The end of the last sentence is missing?

Reply:
Oh, I didn't remove enough. The official caddy package ships with two services - caddy.service and caddy-api.service. The last sentence was a note to use the caddy-api service to make live config changes.